pkg/controller/platform/platform.go

```diff
@@ -35,8 +36,9 @@ func GetType(infraStatus *configv1.InfrastructureStatus) configv1.PlatformType {
 	return infraStatus.Platform
 }
 
-func getClient() (client.Client, error) {
+func getClient(explicitKubeconfig string) (client.Client, error) {
```

Comment:
Should this handle empty string? It seems like this flag won't always be set.

Reply:
a few lines down where we
```rules.ExplicitPath = explicitKubeconfig```
will take the empty string and reassign the empty string to rules.ExplicitPath (which is already the default setting). so in that case passing the empty string to getClient() gets us back the old behavior of not setting ExplicitPath

Reply:
Sounds good. thanks.